Lib/test/test_venv.py

```diff
@@ -25,13 +25,6 @@
 except ImportError:
     ctypes = None
 
-# Platforms that set sys._base_executable can create venvs from within
-# another venv, so no need to skip tests that require venv.create().
-requireVenvCreate = unittest.skipUnless(
-    hasattr(sys, '_base_executable')
-    or sys.prefix == sys.base_prefix,
-    'cannot run venv.create from within a venv on this platform')
-
```

Comment:
I need to do some work to set `base_executable` properly on Linux to make this work, so I may just undo it and leave nested-venv creation as unsupported.